src/realm_dart_sync.cpp

```diff
@@ -109,7 +109,7 @@ RLM_API void realm_dart_sync_wait_for_completion_callback(realm_userdata_t userd
 
     auto ud = reinterpret_cast<realm_dart_userdata_async_t>(userdata);
     ud->scheduler->invoke([ud, error = std::move(error_copy)]() {
-        (reinterpret_cast<realm_sync_download_completion_func_t>(ud->dart_callback))(ud->handle, error.get());
+        (reinterpret_cast<realm_sync_wait_for_completion_func_t>(ud->dart_callback))(ud->handle, error.get());
```

Comment:
Not sure I followed the changes correctly to here. Was this a bug or something else?

Reply:
This is a change in the name from bumping the Core version.